site/troubleshooting.md

```diff
@@ -275,19 +275,13 @@ re-connect.
 
 ## <a name="reboots"></a>Reboots
 
-because the process of getting everything re-started and
-re-connected via restart policies is not entirely reliable, the router and 
-proxy containers do not have Docker restart policies
-set. Until that changes, we recommend that you create appropriate startup scripts to launch
-Weave Net and run application containers from
-[your favourite process manager](/site/systemd.md).
+All the containers started by `weave launch` are configured with
+Docker restart policy `--restart=always`, so they will come back again
+on reboot.
 
-If you are shutting down or restarting a host deliberately, run `weave
-reset` to clear everything down.
-
-The Weave Net Docker plugin does restart automatically because it must
-always start with Docker, as described in
-[its documentation](/site/weave-docker-api/using-proxy.md).
+Note that `weaveproxy` is responsible for re-attaching everything at
```

Comment:
"...responsible for reconfiguring the Weave router and re-attaching application containers to the Weave network".